start-linux-watchdog.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
Laten we alle "controles" op aanwezige executables naar het startup script verplaatsen aangezien ik vermoed dat de foutmeldingen die uit dit script volgen nog minder "vindbaar" zullen zijn dan fouten die dat script produceert. Omdat een deel van het script pas na een zekere wachttijd wordt uitgevoerd komt de melding mogelijk ook pas later (heb die code nog niet bekeken dus mogelijk wordt de check wel meteen gedaan).